test/browser/spider.js

```diff
@@ -19,6 +25,20 @@ function findSrcSetURLs(resourceText, baseURL) {
   return urls;
 }
 
+function findOpenGraphURLs(resourceText, baseURL) {
+  var urls = [];
+
+  resourceText.replace(/\sproperty="og:image"\s?content\s?=\s?['"]([^"']+)/ig, function() {
+    arguments[1].split(',').forEach(function(resource) {
```

Comment:
Oh, sorry, all I wanted to do was find all the occurrences of a RegExp in a string and get an array of all their [capturing groups](http://www.regular-expressions.info/brackets.html)--so that I could find what the value of the `content` attribute is--but there appears to be no way to easily do that aside from calling `replace` and passing it a function. The first argument passed to the function is the whole string, i think, while the following arguments are the capturing groups, or something. It's all ridiculously complicated and not well-documented. :sob: